compiler/prelude/regen.go

```diff
@@ -0,0 +1,34 @@
+// +build ignore
+
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/gopherjs/gopherjs/compiler/prelude"
+)
+
+func main() {
+	cmd := exec.Command("uglifyjs")
+	cmd.Stdin = strings.NewReader(prelude.Prelude)
+
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		log.Fatalf("uglifyjs: %v\n%s", err, out)
+		os.Exit(1)
```

Comment:
Calling `os.Exit(1)` after `log.Fatalf` is a no-op, it will never run.

Can you refactor the code to follow this pattern?

```Go
package main

import (
	"log"
)

func main() {
	err := run()
	if err != nil {
		log.Fatalln(err)
	}
}

func run() error {
	// move the code that can fail here
	if err != nil {
		return nil
	}
	// ...
	return nil
}

Reply:
Done